lib/pbench/server/api/resources/query_apis/index_mappings.py

```diff
@@ -81,5 +81,5 @@ def get(self, index_name: AnyStr) -> Dict[AnyStr, Any]:
             # construct response object
             return jsonify(result)
         except TemplateNotFound:
-            self.logger.exception("Run document template not found in the database")
+            self.logger.exception("{} Run document template not found in the database", index_name)
```

Comment:
So, `result Run document template`? How about `"Document template {} not found in the database."`??

Reply:
Fixed